2016/Day 6-1.py

```diff
@@ -0,0 +1,10 @@
+from collections import Counter
+
+def extractFile(file):
+    return [line for line in file]
+
+with open('Day 6-Input') as file:
+    lines = extractFile(file)
+    lines = [[char for char in line[:-1]] for line in lines]
```

Comment:
No need for this. Just use `line.strip() for line in file` in the `extractFile` function
Also please follow [pep8](https://www.python.org/dev/peps/pep-0008/) and make function names snake_case